components/asp/assembler/TraceAssembler.py

```
@@ -0,0 +1,168 @@
+        for index, columnValue in enumerate(data):
+            obj[columns[index][1]] = columnValue
+
+        return obj
+
+
+    def processDatabase(self):
+        '''
+            Processes the database and extracts all the traces.
+        '''
+        # Get all the start nodes
+        query = f'SELECT * FROM "IOEVENTS" WHERE "type" = ?' 
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)